CHANGELOG.md

```diff
@@ -22,15 +22,27 @@ If possible, provide tooling that performs the changes, e.g. a shell-script.
 ## New features
 
 * Traits for "metaprogramming" with `seqan3::type_list` and type packs have been added.
+* Support for PowerPC has been added.
```

Comment:
I think we should make an extra section for `## Platform support`. 

Also please specify which kind of PowerPC. This is little endian ppc64 or did we also test big endian?

I am reluctant about adding offical platform support that we have no CI for, it could break with any commit.

Reply:
Agreed. Let's postpone it until we have CI.